app/src/rgb_underglow.c

```diff
@@ -27,8 +27,14 @@
 
 LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
-#define STRIP_LABEL DT_LABEL(DT_CHOSEN(zmk_underglow))
-#define STRIP_NUM_PIXELS DT_PROP(DT_CHOSEN(zmk_underglow), chain_length)
+#if !DT_CHOSEN(zmk_underglow)
```

Comment:
Is a previous commit, you changed DT_CHOSEN() to DT_HAS_CHOSEN(). Should this be the same?